From 0070081ffe65a4fbe442044ddfcc818593c98e33 Mon Sep 17 00:00:00 2001
From: Romain Naour <romain.naour@openwide.fr>
Date: Wed, 30 Oct 2013 22:35:34 +0100
Subject: [PATCH] Fix ptrace detection

Ptrace support detection is ignored when CFLAGS is set in command
arguments. Use override in Makefile to add -DWARN_NO_PTRACE and
-DNO_PTRACE in CFLAGS if ptrace is not available.

Signed-off-by: Romain Naour <romain.naour@openwide.fr>
---
 Makefile | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/Makefile b/Makefile
index 83329ca..01b4f45 100644
--- a/Makefile
+++ b/Makefile
@@ -223,12 +223,12 @@ LDFLAGS ?=
 ifndef NO_PTRACE
 ifneq ($(call try-cc,$(SOURCE_PTRACE),),y)
 	NO_PTRACE = 1
-	CFLAGS += -DWARN_NO_PTRACE
+	override CFLAGS += -DWARN_NO_PTRACE
 endif
 endif
 
 ifdef NO_PTRACE
-CFLAGS += -DNO_PTRACE
+override CFLAGS += -DNO_PTRACE
 endif
 
 # Append required CFLAGS
-- 
1.8.1.2

